src/main/java/com/google/cloud/teleport/templates/PubsubToText.java

```diff
@@ -147,14 +199,24 @@ public static PipelineResult run(Options options) {
     // Create the pipeline
     Pipeline pipeline = Pipeline.create(options);
 
+    ValueProvider<String> inputSubscription = options.getInputSubscription();
```

Comment:
Extra assignment statements inputSubscription, inputTopic can be eliminated

Reply:
👍